internal/services/topic.go

```diff
@@ -0,0 +1,171 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package services defines some canary related services
+package services
+
+import (
+	"log"
+	"math"
+	"os"
+	"strconv"
+
+	"github.com/Shopify/sarama"
+	"github.com/strimzi/strimzi-canary/internal/config"
+)
+
+type TopicService interface {
+	Reconcile() (int, map[int32][]int32, bool, error)
+	Close()
+}
+
+type topicService struct {
+	canaryConfig *config.CanaryConfig
+	client       sarama.Client
+	admin        sarama.ClusterAdmin
+}
+
+func NewTopicService(canaryConfig *config.CanaryConfig, client sarama.Client) TopicService {
+	/*
+		adminConfig := sarama.NewConfig()
+		adminConfig.Version = sarama.V2_6_0_0
+		admin, err := sarama.NewClusterAdmin([]string{canaryConfig.BootstrapServers}, adminConfig)
+	*/
```

Comment:
is commented code left here as a reminder of what config init looks like or is it an artifact of the refactor that can be removed?

Reply:
It's a leftover ... removing it, thanks!